docs/source/user-guide/tuning.md

```diff
@@ -141,30 +191,22 @@ It must be set before the Spark context is created. You can enable or disable Co
 at runtime by setting `spark.comet.exec.shuffle.enabled` to `true` or `false`.
 Once it is disabled, Comet will fall back to the default Spark shuffle manager.
 
-### Shuffle Mode
+### Shuffle Implementations
 
-Comet provides three shuffle modes: Columnar Shuffle, Native Shuffle and Auto Mode.
+Comet provides two shuffle implementations: Native Shuffle and Columnar Shuffle. Comet will use Native Shuffle 
+where possible, then will use Columnar Shuffle where possible, and will fall back to Spark for shuffle operations 
+that cannot be supported by either. 
 
-#### Auto Mode
```

Comment:
Should we keep the explanation how the auto mode works? e.g. native -> columnar -> spark

Reply:
This is now explained under `### Shuffle Implementations` based on Parth's suggested text.